.gitignore

```diff
@@ -2,4 +2,5 @@
 .DS_Store
 .vscode/
 node_modules/
-yarn.lock
\ No newline at end of file
+yarn.lock
```

Comment:
We need the `lock` file to rebuild the same

Reply:
Actually `yarn.lock` it's already been ignored. 🤔 

I think that's because it has `package.lock.json`

Reply:
Actually your branch is behind master. It has been changed in the latest commit